solution/ui/regulations/eregs-component-lib/src/components/RecentChangesContainer.vue

```diff
@@ -89,14 +89,3 @@ export default {
         />
     </div>
 </template>
-
-<style lang="scss">
```

Comment:
we're moving away from `<style>` tags included in a Single File Component; these style rules are instead declared in `.scss` files with the rest of our style rules.